pset-functions.js

```diff
@@ -90,6 +107,23 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+function wordCount(phrase) {
+  let words = phrase.split(" ")
+  let wordsMap = {}
+  words.forEach(function(key)  {
+    if (wordsMap.hasOwnProperty(key)) {
+      wordsMap[key]++
+    } else {
+      wordsMap[key] = 1
+    }
+  })
+  let keys = Object.keys(wordsMap)
+  for (key of keys) {
+    console.log(key + ":" + " " + wordsMap[key])
+  }
+}
+
+wordCount("Humpty Dumpty sat on a wall Humpty Dumpty had a great fall")
```

Comment:
@laraann88 Great work coming up with this solution here. I liked how you used the `Object` method hasOwnProperty to check to see if the word that is passed exist as a key in that object. if true increment by 1 if false create an object key and set to 1
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L103)